### PR TITLE
Azure release

### PR DESCRIPTION
**Pull request by lzchen**
Cont. from [#699]

### CODE REVIEW

**Comment by lzchen** [comment]
image
Getting merge failure when attempting to merge [#699], attempting to fix.

File: contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+                        )
+                    return
+                except Exception:
+                    logger.exception(
+                        'Error while processing %s: %s.',
+                        status_code,
+                        text
+                    )
+                    return
+        # Check for non-tryable result
```
Comment (reyang) [inline contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py]: ‘non-retryable’

Comment (lzchen) [reply contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py]: Nice catch.

**Comment by lzchen** [comment]
@c24t  @songy23  Can one of you attempt to merge this PR? @reyang is getting a merge failed error when he attempts to do it.

If it still fails, can we disable the CLA check temporarily for this merge?

**Comment by songy23** [comment]
Will do, PR checks look all good to me though:

Screen Shot 2019-07-23 at 3 29 04 PM

**Comment by songy23** [comment]
OK now I see the failure:
Screen Shot 2019-07-23 at 3 30 59 PM

I think this has something to do with the branch protection rule since I haven't seen similar issue when merging to master. @c24t can you disable the signature check on v0.6.x?

**Comment by c24t** [comment]
I disabled requiring signed commits for the release branches and restricted pushing to repo admins.

**Pull request merged**